.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+Feature requests are not handled in the repository. The best place to discuss these ideas would be on the /tg/station 13 forums here: https://tgstation13.org/phpBB/viewforum.php?f=9&sid=5153c1c704a4fb1006bf7a265e45e03f
```

Comment:
Good day @kevinz000, I'm fairly certain a citadel's dev forum existed, but couldn't remember. Would you be kind and link it? 
Need to make this line more consistent with the community. 
~~Would die inside if I were to see some naive post a blunder on this.~~